projects/packages/videopress/src/client/block-editor/blocks/videopress/components/videopress-uploader/index.js

```diff
@@ -271,6 +340,10 @@ const VideoPressUploader = ( { attributes, setAttributes, noticeUI, noticeOperat
 		return errorMessage;
 	};
 
+	if ( isLoading ) {
+		return <span>Loading...</span>;
```

Comment:
I believe this should be localized

Reply:
This is just a placeholder for now... there are many states in the block that still needs to be properly designed